src/index.html

```diff
@@ -9,6 +9,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <div class="logo"><img src="./images/logo.png" alt="logo"></div>
```

Comment:
```suggestion
        <div class="logo">
          <img src="./images/logo.png" alt="logo">
        </div>
```
Новый тег - новая строка